manifests/kots-config.yaml

```diff
@@ -263,10 +265,10 @@ spec:
           type: text
           required: true
           default: "carto-onprem.lan"
-          validation:
-            regex:
-              pattern: ^([a-zA-Z0-9_\\-]+\\.)+[a-zA-Z]{2,}(:[0-9]+)?$
-              message: "The domain must be only a domain/subdomain. We don't support paths or protocols."
+          # validation:
```

Comment:
I have the fix here https://github.com/CartoDB/carto-selfhosted-helm/pull/478 so we should merge it before next release